libs/ui/src/link_button.tsx

```diff
@@ -1,38 +1,52 @@
 import { useHistory } from 'react-router-dom';
-import { Button, ButtonProps } from './button';
+import { forwardRef, Ref, SyntheticEvent } from 'react';
+import { Button, ButtonProps, ClickHandler, TypedClickHandler } from './button';
 
 /**
  * Props for {@link LinkButton}.
  */
 export interface LinkButtonProps extends Omit<ButtonProps, 'onPress'> {
+  // export type LinkButtonProps = ButtonProps & {
   goBack?: boolean;
   primary?: boolean;
   to?: string;
+  ref?: Ref<Button<SyntheticEvent>>;
+  // onPress not optional for <Button> but optional here. Its behavior
+  onPress?: ClickHandler | TypedClickHandler<unknown>;
```

Comment:
This discards the feature of `Button` where if `onPress` takes a value `T` then the `value` prop is required.

Reply:
We never actually use `TypedClickHandler` (no existing usage of `LinkButton` passes a value, so there is no case where we want to pass a value through to `Button`). In [f5d6a6b](https://github.com/votingworks/vxsuite/pull/4284/commits/f5d6a6b0e18c065a8aa91b3b043a1a7d71cdac2c) I deleted `TypedClickHandler` from allowable types for `LinkButton.onPress` to make this explicit. This also avoids the problem you described.